### PR TITLE
Sad host yard show

### LINKED ISSUE #33: Host Yard Show Page - Sad Path
If the YS-ENGINE does not respond, maybe display an error and redirect back to the dashboard?

### PR DESCRIPTION
**Pull request by dev-named-jordan**
Adds a sad path test to front end host yard show, for if backend engine fails to give response. 

close this issue #33

### CODE REVIEW

File: spec/features/yards/show_spec.rb [hunk 1]
```diff
@@ -93,4 +93,17 @@
...
     end
   end
+  describe "Sad path for engine failure" do
+    it 'Displays an error, and re-directs when engine errors.' do
+
+      visit yard_path(1)
+
+      error = 'Data cannot be accessed at this time'
+      response = stub_request(:get, "#{ENV['ys_engine_url']}/api/v1/yards/1")
+      .to_return(status: [500, error], headers: {})
```
Comment (amsmyth1) [inline spec/features/yards/show_spec.rb:103]: this code is pretty

**Review by amsmyth1 — APPROVED** [review]
great job!

File: spec/features/yards/show_spec.rb (hunk 1 above)
Comment (Gvieve) [inline spec/features/yards/show_spec.rb:103]: Amazing! Just AMAZING!

**Review by Gvieve — APPROVED** [review]
This looks fantastic. Great job all!

**Pull request merged**